src/app/feed/feed.component.html

```diff
@@ -3,7 +3,7 @@
 				[searchInputControl]="_queryControl"
 				(searchEvantEmitter)="handleSearchQuery()"></feed-header>
 
-	<div class="searching">
+	<div *ngIf="(isSearching$ | async)" class="searching" style="margin-top: 10%">
```

Comment:
I removed inline styles from the mentioned file. I'll make them committed to the same pull request. @hemantjadon what do you mean by unable to see the animation. What is shown to you when you hit enter after entering key word? Please clarify.